test/Microsoft.DotNet.ProjectJsonMigration.Tests/GivenAProjectMigrator.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.Build.Construction;
+using Microsoft.DotNet.ProjectJsonMigration.Rules;
+using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.Tools.Common;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using NuGet.Frameworks;
+using Xunit;
+
+namespace Microsoft.DotNet.ProjectJsonMigration.Tests
+{
+    public class GivenAProjectMigrator : TestBase
+    {
+        [Fact]
+        public void It_copies_ProjectDirectory_contents_to_OutputDirectory_when_the_directories_are_different()
```

Comment:
This test ensures that when output directory is set, the project source files and other contents are copied over.